### PR TITLE
Fork pgbackups

### PR DESCRIPTION
**Pull request by neovintage**
Transfer command fixed.  All databases copy correctly. 

/cc @dickeyxxx

### CODE REVIEW

**Comment by uhoh-itsmaciek** [comment]
[This](https://github.com/heroku/heroku/blob/fork-pgbackups/lib/heroku/command/fork.rb#L130-L136) should go, along with its call sites.

File: lib/heroku/command/fork.rb [hunk 1]
```diff
@@ -144,20 +144,22 @@ def migrate_db(from_addon, from, to_addon, to)
...
         to_attachment = to_addon["message"].match(/Attached as (\w+)_URL\n/)[1]
 
-        pgb = Heroku::Client::Pgbackups.new(from_config["PGBACKUPS_URL"])
-        transfer = pgb.create_transfer(
+        resolver = Heroku::Helpers::HerokuPostgresql::Resolver.new(to, api)
+        attachment = resolver.resolve("#{to_attachment}_URL", nil)
+        pgb = Heroku::Client::HerokuPostgresql.new(attachment)
+        transfer = pgb.pg_copy(
+          from_attachment.gsub('HEROKU_POSTGRESQL_',''),
           from_config["#{from_attachment}_URL"],
```
Comment (uhoh-itsmaciek) [inline lib/heroku/command/fork.rb]: So I'm not sure about the soundness of getting the URL like this, but you're not changing that aspect of this functionality, so I think it's fine for now--it's just something we may want to keep in mind, especially with the shareable add-on changes.

**Comment by uhoh-itsmaciek** [comment]
+1, but it looks like you already merged, @dickeyxxx?

**Comment by dickeyxxx** [comment]
yeah, I tested it and it seemed to work. I do agree with you the way it gathers the URL isn't ideal though

**Comment by dickeyxxx** [comment]
but as you pointed out, it was like that before

**Comment by neovintage** [comment]
I agree.  This was meant as more of stop-gap to get pgbackups using
transferatu.

**Comment by uhoh-itsmaciek** [comment]
Ok, great. Thanks.

**Pull request merged**